### DISCUSSION TITLE
Fix the description of "AWS Allowed Resource Types" on the help page below

### DISCUSSION THREAD

**Discussion by Tringga** [QualiTorque/qualitorque.github.io#136] [Documentation requests] [answered]
https://qualisystemscom.zendesk.com/agent/tickets/52837   
 Please fix the description of "AWS Allowed Resource Types" on the help page below.
https://docs.qtorque.io/admin-guide/governance/policies#available-policies _[steve] updated link_
I understand that the resources that can be specified in the "AWS Allowed Resource Types" policy are those with "aws_" at the beginning.
In that case, the following link in the red frame of the attached image (AWS Allowed Resource Types.png) may misunderstand users. https://registry.terraform.io/providers/hashicorp/aws/latest/docs#skip_requesting_account_id
The list in the link above is only partial and users may misunderstand that they can only specify the resources in the list.
Therefore, I would like you to modify it to say, "The resources that can be specified are those with "aws_" at the beginning."

**Comment by Shnitselon** [comment, marked as answer]
@Tringga thank you so much for the feedback.
Updated as requested :)

**Reply by Tringga** [reply]
@Shnitselon

I would like to request a re-update with the following content.
"The allowed resource types are those beginning with "aws_" here."
Regarding the above sentence, I would like you to delete the "here" link and correct it to the following sentence.
"The allowed resource types are those beginning with "aws_". For example, aws_vpc, aws_instance, etc."

Can you change the above per customer request?

**Reply by Shnitselon** [reply]
@Tringga updated as requested
